nengo/transforms.py

```diff
@@ -42,8 +44,7 @@ class Transform(FrozenObject):
     """A base class for connection transforms."""
 
     def sample(self, rng=np.random):
-        """
-        Returns concrete weights needed to implement the specific transform.
+        """Returns concrete weights to implement the specified transform.
```

Comment:
I was actually using some automated parser at some point that didn't parse the docstrings properly if they didn't start on the next line (which is why I started doing it that way).  I don't remember what that parser was in order to demonstrate though 🤷‍♂️ .

Reply:
Hm, interesting. There's a lot of docstrings we'll have to change if that's the case so I say we keep it consistent for now and if we find a tool that doesn't parse it correctly we can switch.